x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvPSeriesWeightedSumTests.java

```diff
@@ -55,7 +55,7 @@ private static void doubles(List<TestCaseSupplier> cases) {
                 ),
                 "MvPSeriesWeightedSumDoubleEvaluator[block=Attribute[channel=0], p=" + p + "]",
                 DataType.DOUBLE,
-                closeTo(calcPSeriesWeightedSum(field, p), 0.00000001)
+                closeTo(calcPSeriesWeightedSum(field, p), .00001)
```

Comment:
I wonder if this should be instead something like:
```suggestion
                closeTo(expectedResult, expectedResult * .00000001)
```
As Double precision depends on its exponent, and from what I see, [Hamcrest](https://github.com/hamcrest/JavaHamcrest/blob/master/hamcrest/src/main/java/org/hamcrest/number/IsCloseTo.java#L46) doesn't take care of it.

Reply:
I think that's probably a good call, yeah.